packages/cli/src/commands/optimize/optimize.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @flow
+ */
+
+const fs = require('fs').promises;
+const path = require('path');
+import sharp from 'sharp';
+import type {ConfigT} from 'types';
+import optimizeCommandLineArgs from './optimizeCommandLineArgs';
+import {logger} from '@react-native-community/cli-tools';
+
+function isImage(filePath: string) {
+  const regex = /\.(png|jpg|jpeg)/;
```

Comment:
This regex is very permissive. Please use something like `/\.png|jpg|jpeg$/i` (notice `$` to mark end of the string and added `i` to avoid lowercasing later)

Reply:
added **$** for regex